### ISSUE TITLE
[E893] Could not find function 'customize_language_data' in function registry 'callbacks'

### ISSUE THREAD

**Issue by dinogowda** [explosion/spaCy#8668]
### Discussed in https://github.com/explosion/spaCy/discussions/8667

<div type='discussions-op-text'>

<sup>Originally posted by **dinogowda** July  9, 2021</sup>
Hi Team 

I have custom registry function "customize_language_data" declared in function.py which is being used in "nlp.before_creation" Spacy CLI training. This helps update language defaults. After training completion i attempted to load model-best and faced below error.  Any idea on how to resolve this ?

"[E893] Could not find function 'customize_language_data' in function registry 'callbacks'". 

Below is my Config and file and also funcion.py( Custom code)

CONFIG.CFG
========
[paths]
train = null
dev = null
vectors = null
init_tok2vec = null

[system]
gpu_allocator = "pytorch"
seed = 0

[nlp]
lang = "en"
pipeline = ["transformer","ner"]
batch_size = 128
disabled = []
#before_creation = null
after_creation = null
after_pipeline_creation = null
tokenizer = {"@tokenizers":"spacy.Tokenizer.v1"}

[nlp.before_creation]
@callbacks= "customize_language_data"

[components]

[components.ner]
#factory = "ner"
source = null
moves = null
update_with_oracle_cut_size = 100

[components.ner.model]
@architectures = "spacy.TransitionBasedParser.v2"
state_type = "ner"
extra_state_tokens = false
hidden_width = 64
maxout_pieces = 2
use_upper = false
nO = null

[components.ner.model.tok2vec]
@architectures = "spacy-transformers.TransformerListener.v1"
grad_factor = 1.0
pooling = {"@layers":"reduce_mean.v1"}
upstream = "*"

[components.transformer]
#factory = "transformer"
source = null
max_batch_items = 4096
set_extra_annotations = {"@annotation_setters":"spacy-transformers.null_annotation_setter.v1"}

[components.transformer.model]
@architectures = "spacy-transformers.TransformerModel.v1"
name = "roberta-base"

[components.transformer.model.get_spans]
@span_getters = "spacy-transformers.strided_spans.v1"
window = 128
stride = 96

[components.transformer.model.tokenizer_config]
use_fast = true

[corpora]

[corpora.dev]
@readers = "spacy.Corpus.v1"
path = ${paths.dev}
max_length = 0
gold_preproc = false
limit = 0
augmenter = null

[corpora.train]
@readers = "spacy.Corpus.v1"
path = ${paths.train}
max_length = 500
gold_preproc = false
limit = 0
augmenter = null

[training]
accumulate_gradient = 3
dev_corpus = "corpora.dev"
train_corpus = "corpora.train"
seed = ${system.seed}
gpu_allocator = ${system.gpu_allocator}
dropout = 0.4
patience = 4000
max_epochs = 3
max_steps = 20000
eval_frequency = 200
frozen_components = []
before_to_disk = null

[training.batcher]
@batchers = "spacy.batch_by_padded.v1"
discard_oversize =  false
size = 2000
buffer = 256
get_length = null

[training.logger]
@loggers = "spacy.ConsoleLogger.v1"
progress_bar = True

[training.optimizer]
@optimizers = "Adam.v1"
beta1 = 0.9
beta2 = 0.999
L2_is_weight_decay = true
L2 = 0.01
grad_clip = 1.0
use_averages = false
eps = 0.00000001

[training.optimizer.learn_rate]
@schedules = "warmup_linear.v1"
warmup_steps = 250
total_steps = 20000
initial_rate =  0.000005

[training.score_weights]
ents_per_type = null
ents_f = 0.2
ents_p = 0.3
ents_r = 0.5

[pretraining]

[initialize]
vectors = null
init_tok2vec = ${paths.init_tok2vec}
vocab_data = null
lookups = null
before_init = null
after_init = null

#Custome code added for Manual TOkenization
#[initialize.before_init]
#@callbacks = "customize_language_data"

[initialize.components]

[initialize.tokenizer]

Function.py
=====

@spacy.registry.callbacks("customize_language_data")
def create_callback():
    def customize_language_data(lang_cls):
        lang_cls.Defaults.prefixes.extend(token_symbols)
        lang_cls.Defaults.suffixes.extend(token_symbols)
        lang_cls.Defaults.infixes.extend(token_symbols)
        return lang_cls
    return customize_language_data

</div>

**Comment by adrianeboyd** [comment]
I'm not sure how this duplicate issue got created? Let's keep this in the linked discussion thread...
